client/src/components/InputField/InputField.js

```diff
@@ -1,12 +1,34 @@
 import React, { Component } from 'react';
-import { Input } from 'reactstrap';
+import {
+  Button,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  Popover,
+  PopoverBody,
+} from 'reactstrap';
 import PropTypes from 'prop-types';
 
+const tipImportanceColorMap = {
+  normal: 'muted',
+  high: 'danger',
+};
+
+const tipImportanceMessageMap = {
+  normal: 'Tip',
+  high: 'Important tip',
+};
+
```

Comment:
Why not combine there two objects into one? The keys probably need to be const and the message need to be translatable.
```suggestion
export const TIP_IMPORTANCES = {
  NORMAL: 'normal',
  HIGH: 'danger'
}

const tipMap = {
  [TIP_IMPORTANCES.NORMAL]: {
    color: 'muted',
    message: i18n._t('Admin.TIP_IMPORTANCE_NORMAL', 'Tip')
  },
  [TIP_IMPORTANCES.HIGH]: {
    color: 'danger',
    message: i18n._t('Admin.TIP_IMPORTANCE_HIGH', 'Important tip')
};

const tipImportanceMessageMap = {
  normal: 'Tip',
  high: 'Important tip',
};
```

Reply:
Excellent point - I've implemented this change in the new `Tip` component.